proto/metadata.proto

```diff
@@ -44,6 +44,11 @@ message Metadata {
     ondatra.Device.Vendor vendor = 1;
     // Hardware models of the device.
     repeated string hardware_model = 2;
+    // Regex for hardware model of the device.
```

Comment:
Add a comment on the new fields that leaving them empty matches any {hardware model, software version}.

Reply:
done

Reply:
Instead, I would include a unique line per comment, i.e.

```
    // Regex for hardware model of the device.
    // The empty string will match any hardware model.
    string hardware_model_regex = 3;
    // Regex for software version of the device.
    // The empty string will match any software version.
    string software_version_regex = 4;
```

Reply:
Done